OpenAPI/kiota/using.md

```diff
@@ -236,9 +237,17 @@ Enables backing store for models. Defaults to `false`.
 kiota generate --backing-store
 ```
 
+### `--exclude-backward-compatible (--ebc)`
+
+Whether to exclude the code generated only for backward compatibility reasons or not. Defaults to `false`.
```

Comment:
It's not clear what exactly this means. I read through the linked PR and issue, and it sounds like there's new code in Kiota to generate extra classes to keep the old behavior, and this flag would skip that? From a user perspective, it's hard for me to understand if I should use this flag or not. Is this something that could be concisely explained in this topic?

Reply:
added an abstract, let me know whether it clarifies things